coordinator/src/step_3.rs

```diff
@@ -0,0 +1,72 @@
+use frost::{round2::SignatureShare, Identifier, Signature, SigningPackage};
+
+use frost_ed25519 as frost;
+
+use std::{
+    collections::HashMap,
+    io::{BufRead, Write},
+};
+
+use crate::step_1::ParticipantsConfig;
+
+pub fn step_3(
+    input: &mut impl BufRead,
+    logger: &mut dyn Write,
+    participants: ParticipantsConfig,
+    signing_package: SigningPackage,
+) {
+    let group_signature =
+        request_inputs_signature_shares(input, logger, participants, signing_package).unwrap();
+    print_signature(logger, group_signature);
+}
+
+// Input required:
+// 1. number of signers (TODO: maybe pass this in?)
+// 2. signatures for all signers
+fn request_inputs_signature_shares(
+    input: &mut impl BufRead,
+    logger: &mut dyn Write,
+    participants: ParticipantsConfig,
+    signing_package: SigningPackage,
+) -> Result<Signature, Box<dyn std::error::Error>> {
+    writeln!(
+        logger,
+        "The number of signers: {}",
+        participants.participants.len()
+    )?;
+    println!("{}", participants.participants.len());
+
```

Comment:
Is this needed?

```suggestion
```

Reply:
Probably not. I'll remove

Reply:
Done in [9f2c617](https://github.com/ZcashFoundation/frost-zcash-demo/pull/59/commits/9f2c617dab141cf89d37d92d0e0ef5f8d14ff537)